src/plugins/aws/ssh.ts

```diff
@@ -87,19 +88,19 @@ export const awsSshProvider: SshProvider<
 
   preTestAccessPropagationArgs: () => undefined,
 
-  proxyCommand: (request) => {
+  proxyCommand: (request, port) => {
     return [
       "aws",
       "ssm",
       "start-session",
       "--region",
       request.region,
       "--target",
-      "%h",
+      request.id,
```

Comment:
why do you need to specify `request.id` here? wouldn't `%h` be enough since we add the hostname to the config?